docs/guides/support/testing.md

```
@@ -0,0 +1,428 @@
+# Run single test
+pytest tests/test_manager.py::TestAIPipelineManager::test_generate_image
+
+# Run with verbose output
+pytest tests/ -v
+```
+
+---
+
+## Test Organization
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)